From 4bf351fb0292f2e12494221523bf76e2948bea7c Mon Sep 17 00:00:00 2001
From: Adhemerval Zanella <adhemerval.zanella@linaro.org>
Date: Mon, 28 Mar 2022 14:55:52 -0300
Subject: [PATCH 038/163] localedate: Fix printf type on tst_mbrtowc

Checked on x86_64-linux-gnu and i686-linux-gnu.

(cherry picked from commit 74942fd2735afaadf675b38abc19c65288155b02)
---
 localedata/tests-mbwc/tst_mbrtowc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localedata/tests-mbwc/tst_mbrtowc.c b/localedata/tests-mbwc/tst_mbrtowc.c
index 20507abd99..94bd28aba8 100644
--- a/localedata/tests-mbwc/tst_mbrtowc.c
+++ b/localedata/tests-mbwc/tst_mbrtowc.c
@@ -64,7 +64,7 @@ tst_mbrtowc (FILE * fp, int debug_flg)
 	  {
 	    fprintf (stdout, "mbrtowc() [ %s : %d : %d ] ret = %zd\n",
 		     locale, rec + 1, seq_num + 1, ret);
-	    fprintf (stdout, "			    errno = %hd\n",
+	    fprintf (stdout, "			    errno = %d\n",
 		     errno_save);
 	  }
 
-- 
2.37.2

